tests/test_analysis.py

```diff
@@ -0,0 +1,912 @@
+import unittest
+from collections import defaultdict
+from copy import copy
+from pathlib import Path, PosixPath
+
+import pandas as pd
+
+from pheval.analyse.analysis import (
+    AssessGenePrioritisation,
+    AssessVariantPrioritisation,
+    GenePrioritisationResultData,
+    PrioritisationRankRecorder,
+    RankComparisonGenerator,
+    RankStats,
+    VariantPrioritisationResultData,
+    merge_results,
+)
+from pheval.utils.phenopacket_utils import GenomicVariant, ProbandCausativeGene
+
+
+class TestPrioritisationRankRecorder(unittest.TestCase):
+    def setUp(self) -> None:
+        self.add_new_phenopacket_variant_record = PrioritisationRankRecorder(
+            1,
+            Path("directory1"),
+            VariantPrioritisationResultData(
+                Path("/path/to/phenopacket-2.json"), GenomicVariant("1", 4896347, "C", "T"), 9
```

Comment:
These "hypothetical" paths will work during the execution of the test?

Reply:
Yep, these "hypothetical" paths will work as the path in this case isn't being used to open/locate a file, it is printed to one of the output files which detail the rank given to specific genes/variants for a phenopacket.